### PR TITLE
Update details for Vercel

### PR DESCRIPTION
**Pull request by PonomareVlad**

### CODE REVIEW

**Comment by PonomareVlad** [comment]
### Proofs:
- [ ] Vercel Serverless — https://grammy.dev/hosting/vercel.html
- [ ] Vercel Edge — https://github.com/PonomareVlad/grammYVercelEdge

File: site/docs/hosting/comparison.md [hunk 1]
```diff
@@ -36,10 +36,10 @@ The main thing to know about them is that on serverless infrastructures you are
...
-| Vercel                 | Free       | $20/mo subscription                                                                                        | [Unlimited invocations, 100 GB-h, 10 s time limit](https://vercel.com/pricing)          | ❓      | ❓                           | ❓  | Not intended for non-websites?                                                                   |
+| Vercel                 | Free       | $20/mo subscription                                                                                        | [Unlimited invocations, 100 GB-h, 10 s time limit](https://vercel.com/pricing)          | [✅](https://vercel.com/docs/concepts/functions/serverless-functions/runtimes/node-js)      | [✅](https://github.com/vercel-community/deno)                           | [✅](https://vercel.com/docs/frameworks)  | Only webhooks supported                                                                   |
```
Comment (KnorpelSenf) [inline site/docs/hosting/comparison.md:39]: I think all platforms listed here only support webhooks. There's no need to mention it here.

Comment (PonomareVlad) [reply site/docs/hosting/comparison.md]: Updated

**Review by KnorpelSenf — APPROVED** [review]
Thank you!

**Review by roj1512 — APPROVED** [review]

**Pull request merged**